src/home/scraper/issue-scraper.ts

```diff
@@ -0,0 +1,49 @@
+import { checkSupabaseSession } from "../rendering/render-github-login-button";
+
+export async function startIssueScraper(username: string) {
+  const supabaseAuth = await checkSupabaseSession();
+
+  // Get the last fetch timestamp
+  const lastFetchKey = `lastFetch_${username}`;
+  const lastFetch = localStorage.getItem(lastFetchKey);
+  const now = Date.now();
+
+  // Prepare the request payload
+  const requestBody: { authToken: string; timestamp?: number } = {
+    authToken: supabaseAuth.provider_token,
+  };
+
+  if (lastFetch) {
+    const lastFetchTimestamp = Number(lastFetch);
+    if (now - lastFetchTimestamp < 24 * 60 * 60 * 1000) {
+      return JSON.stringify({
+        success: true,
+        message: "Skipping fetch - last fetch was less than 24 hours ago",
```

Comment:
The login timeout currently lasts for a day I think, meaning that for 99% of users, they wouldn't retrigger a scrape with a new login in less than 24 hours. Something to note. 

We can also change this timeout in the supabase with settings. Any remarks on what to change it to or leave as is?